### ISSUE TITLE
A potential bug in discrete BCQ FC network

### ISSUE THREAD

**Issue by Trinkle23897** [sfujim/BCQ#6]
Hello, first of all, thanks for this great work!
I found an issue with this line: https://github.com/sfujim/BCQ/blob/9690927c86b9eade8b50f0e69d46a146c4454851/discrete_BCQ/discrete_BCQ.py#L53
I don't think the final output of imitation network logits should be passed by a ReLU layer. If I remove this layer, the code can work on toy examples with the discrete setting.
Also, the convolution network doesn't have the ReLU layer in the imitation logits final output. So I think this should be removed. Please correct me if I'm wrong.

**Comment by sfujim** [comment]
Ah yes, good catch. Thank you! I've corrected this in the code.
